tests/AdaptiveExpressions.Tests/ExpressionParserTests.cs

```diff
@@ -694,7 +694,7 @@ public class ExpressionParserTests
             // init dateTime: 2018-03-15T13:00:00Z
             Test("isDefinite('helloworld')", false),
             Test("isDefinite('2012-12-21')", true),
-            Test("isDefinite('xxxx-12-21')", false),
+            Test("isDefinite('XXXX-12-21')", false),
```

Comment:
>XXXX [](start = 30, length = 4)

Should this actually be case sensitive?

Reply:
Yes, I tested it.  new TimexProperty('xxxx-12-21') will return a empty TimexProperty object.